release/release_tests.yaml

```diff
@@ -5384,6 +5384,49 @@
         cluster_env: stress_tests/stress_tests_app_config.yaml
         cluster_compute: stress_tests/smoke_test_compute_gce.yaml
 
+
+- name: stress_test_many_runtime_envs
+  group: core-daily-test
+  working_dir: nightly_tests
+
+  python: "3.8"
+  frequency: nightly
+  team: core
+  cluster:
+    byod: {}
+    cluster_env: stress_tests/stress_tests_app_config.yaml
+    cluster_compute: stress_tests/stress_tests_compute.yaml
```

Comment:
It's probably too expensive for this test. Can you actually remove smoke test and use smoke_test_compute.yaml as the compute template?

Reply:
we can still have 10000 iterations.

Reply:
done.